docs/source/user_guide.rst

```diff
@@ -0,0 +1,10 @@
+==========
+User Guide
+==========
+
+.. toctree::
+    :glob:
+    :maxdepth: 2
```

Comment:
For this page and examples.rst, let's reduce the max depth and add some intro text so the page isn't just a table of contents